e2e/tests/example.spec.ts

```diff
@@ -0,0 +1,19 @@
+import {test} from '@playwright/test';
+
+test('text typing works', async ({ page }) => {
+  await page.goto('http://127.0.0.1:8080/e2e/site');
+  await page.evaluate(() => {
+    const textbox = new fabric.Textbox('HELLO');
+    fabricCanvas.add(textbox);
+  });
+
+  await page.click('canvas.upper-canvas', {
+    position: { x: 15, y: 15 },
+    clickCount: 2,
+    delay: 200
+  });
```

Comment:
we would probably need some helpers, like click on something. drag control X.
Becuase the playwright test recorder doesn't record the exact position where it clicks ( maybe we could see if is fixable )